website/content/docs/connect/proxies/envoy.mdx

```diff
@@ -471,33 +475,174 @@ definition](/docs/connect/registration/service-registration) or
   that will be appended to the array of [static
   clusters](https://www.envoyproxy.io/docs/envoy/v1.17.2/api-v3/config/bootstrap/v3/bootstrap.proto#envoy-v3-api-field-config-bootstrap-v3-bootstrap-staticresources-clusters)
   in the bootstrap config. This allows adding custom clusters for tracing sinks
-  for example. For a single cluster just encode a single object, for multiple,
-  they should be comma separated with no trailing comma suitable for
-  interpolating directly into a JSON array inside the braces.
+  for example. For a single cluster just, encode a single object. For multiple
```

Comment:
```suggestion
  for example. Encode a single object for a single cluster. For multiple
```
I'm a little unclear, is this what we mean?

Reply:
```suggestion
  for example. In order to configure a single cluster, specify a single JSON object with the cluster details. For multiple
```

Reply:
This is closer to what this sentence was trying to convey.